<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_principal</class>
 <widget class="QMainWindow" name="ventana_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Observador de archivo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_selecionar_carpeta">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecionar carpeta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_direccion_carpeta">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>99</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin capeta selecionada</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_iniciar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(67, 159, 54);</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_parar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(209, 42, 49)</string>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
